alibi_detect/cd/pytorch/classifier.py

```diff
@@ -134,24 +133,31 @@ def score(self, x: np.ndarray) -> float:
 
         Returns
         -------
-        Drift metric (e.g. accuracy) obtained from out-of-fold predictions from a trained classifier.
+        p-value, and a notion of distance between the trained classifier's out-of-fold performance
+        and that which we'd expect under the null assumption of no drift.
         """
         x_ref, x = self.preprocess(x)
+        n_ref, n_cur = x_ref.shape[0], x.shape[0]
+
         x, y, splits = self.get_splits(x_ref, x)
 
         # iterate over folds: train a new model for each fold and make out-of-fold (oof) predictions
-        preds_oof, idx_oof = [], []
+        preds_oof_list, idx_oof_list = [], []
         for idx_tr, idx_te in splits:
             x_tr, y_tr, x_te = x[idx_tr], y[idx_tr], x[idx_te]
             ds_tr = TensorDataset(torch.from_numpy(x_tr), torch.from_numpy(y_tr))
             dl_tr = DataLoader(ds_tr, **self.dl_kwargs)  # type: ignore
             model = deepcopy(self.model)
-            train_args = [model, nn.CrossEntropyLoss(), dl_tr, self.device]
+            loss_fn = nn.CrossEntropyLoss() if (self.preds_type == 'logits') else nn.NLLLoss()
```

Comment:
Small comment: could this be placed in init, similar to tensorflow and set a `self.loss_fn` attribute?